packages/frontend-2/components/viewer/saved-views/panel/views/Group.vue

```diff
@@ -169,7 +169,8 @@ const { on, classes: dropZoneClasses } = useDraggableViewTargetGroup({
       open.value = true
     }
   },
-  enabled: computed(() => !open.value || !viewCount.value)
+  isGroupOpen: computed(() => !!open.value),
```

Comment:
why? aren't both of the drop zones gonna conflict with each other (group zone and view zone)?